src/components/HTMLEngineProvider/index.tsx

```diff
@@ -2,19 +2,12 @@ import React from 'react';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import BaseHTMLEngineProvider from './BaseHTMLEngineProvider';
-import {HTMLEngineProviderProps} from './types';
+import type {HTMLEngineProviderProps} from './types';
 
-function HTMLEngineProvider({debug = false, children = null}: HTMLEngineProviderProps) {
+function HTMLEngineProvider({children = null}: HTMLEngineProviderProps) {
```

Comment:
```suggestion
function HTMLEngineProvider({children}: HTMLEngineProviderProps) {
```

No default value is needed since the prop is required